cmd/webhook/main.go

```diff
@@ -65,9 +68,28 @@ func main() {
 	// Load webhook config
 	c := wh.LoadConfig(*sidecarImage, *imagePullPolicy, *cpuRequest, *cpuLimit, *memoryRequest, *memoryLimit, *ephemeralStorageRequest, *ephemeralStorageLimit)
 
+	// Load config for manager, informers, listers
+	kubeConfig := config.GetConfigOrDie()
+
+	// Setup client
+	client, err := kubernetes.NewForConfig(kubeConfig)
+	if err != nil {
+		klog.Fatalf("Unable to get clientset: %v", err)
+	}
+
+	// Setup stop channel
+	stopCh := signals.SetupSignalHandler()
```

Comment:
We are using this channel for both the informer and the manage. I'm wondering if this is okay, since there are cases in which sharing this is not recommended due to race conditions, but I'm not too familiar.